packages/core/lib/src/data/text_style.dart

```diff
@@ -145,30 +145,30 @@ class TextStyleBuilder<T1> {
     if (_builders == null) return _output = _parentOutput;
 
     _output = _parentOutput?.copyWith(parent: _parentOutput);
-    final l = _builders.length;
+    final l = _builders!.length;
     for (var i = 0; i < l; i++) {
-      final builder = _builders[i];
-      _output = builder(_output, _inputs[i]);
+      final builder = _builders![i];
+      _output = builder(_output, _inputs![i]);
       assert(_output?.parent == _parentOutput);
     }
 
     return _output;
   }
 
   /// Returns `true` if this shares same styling with [other].
-  bool hasSameStyleWith(TextStyleBuilder other) {
+  bool hasSameStyleWith(TextStyleBuilder? other) {
```

Comment:
```suggestion
  bool hasSameStyleWith(TextStyleBuilder other) {
```

Reply:
A null value can be passed from [here](https://github.com/daohoangson/flutter_widget_from_html/pull/436/files#diff-dbf350bf88d9761c4033608f89a5019f5747a9a6f2f1e956600304456671472aR59), one test fails if I make this change.